lib/vsc/install/commontest.py

```diff
@@ -69,6 +69,121 @@
 if sys.version_info >= (3, 5):
     HAS_PROSPECTOR = False
 
+# List of regexps patterns applied to code or message of a prospector.message.Message
+#   Blacklist: if match, skip message, do not check whitelist
+#   Whitelist: if match, fail test
+PROSPECTOR_BLACKLIST = [
+    # 'wrong-import-position',  # not sure about this, these usually have a good reason
+    'Locally disabling',  # shows up when you locally disable a warning, this is the point
+    'Useless suppression',  # shows up when you locally disable/suppress a warning, this is the point
+]
+# to dissable any of these warnings in a block, you can do things like add a comment # pylint: disable=C0321
+PROSPECTOR_WHITELIST = [
+    'undefined',
+    'no-value-for-parameter',
+    'dangerous-default-value',
+    'bare-except',
+    'E713',  # not 'c' in d: -> 'c' not in d:
+    'arguments-differ',
+    'unused-argument',
+    'unused-variable',
+    'reimported',
+    'F811',  # redefinition of unused name
+    'unused-import',
+    'syntax-error',
+    'E101',  # mixing tabs and spaces
+    'bad-indentation',
+    'E111',  # pep8: E111 / indentation is not a multiple of four
+    'bad-whitespace',
+    'trailing-whitespace',
+    'W291',  # pep8: W291 / trailing whitespace
+    #'protected-access',
+    #'logging-not-lazy',
+    'duplicate-key',  # when a key appears twice in a dict definition
+    # prospector cannot check duplicates across files, and pylint only detects duplicates across files (19/03/2019)
+    'duplicate-code', # when 4 or more lines of code show up several times
+    'E501',  # 'line too long'when a line is longer then 120 chars
+    'line-too-long', # use fail using pylint as well (not only pep8 above)
+    # 'protected-access',
+    # 'logging-not-lazy',
+    # will stop working in python3
+    'unpacking-in-except', 'redefine-in-handler',  # except A, B -> except (A, B)
+    'indexing-exception',  # indexing exceptions doesn't work in python3, use Exc.args[index] instead (but why?)
+    'raising-string',  # don't raise strings, raise objects extending Exception
+    'old-octal-literal',  # use 0o700 instead of 0700
+    'import-star-module-level',  # Import * only allowed at module level
+    'old-ne-operator',  # don't use <> as not equal operator, use !=
+    'backtick',  # don't use `variable` to turn a variable in a string, use the str() function
+    'old-raise-syntax',  # sed when the alternate raise syntax raise foo, bar is used instead of raise foo(bar) .
+    'redefined-builtin',
+    # once we get ready to really move to python3
```

Comment:
? the comment suggest to only enable this when "we get ready to really move to python3", so why is this not commented out?